pkg/reconciler/v1alpha1/autoscaling/kpa_scaler.go

```diff
@@ -84,6 +86,46 @@ func (ks *kpaScaler) getAutoscalerConfig() *autoscaler.Config {
 	return ks.autoscalerConfig.DeepCopy()
 }
 
+func scaleBoundInt32(ctx context.Context, kpa *kpa.PodAutoscaler, key string) int32 {
+	logger := logging.FromContext(ctx)
+	if s, ok := kpa.Annotations[key]; ok {
+		i, err := strconv.ParseInt(s, 10, 32)
+		if err != nil || i < 0 {
+			logger.Debugf("value of %s must be positive integer", key)
```

Comment:
I think this should say "... must be a non-negative integer".

Reply:
Why the double negative? :) 

Seriously though, specifying `0` value for these annotations doesn't make much sense. So, I was thinking: what would be the shortest equivalent of "non-negative non-zero integer"

Reply:
I'll change the condition above to make sure `i > 0`. 
Also, changing the message to `... must be an integer greater than 0`

Reply:
Thanks @imikushin.